doc/api/http.md

```diff
@@ -1161,6 +1161,9 @@ it will switch to implicit header mode and flush the implicit headers.
 This sends a chunk of the response body. This method may
 be called multiple times to provide successive parts of the body.
 
+Note that in `http` module, the response body is omitted when the request is a
+HEAD. Similarly, The `204` and `304` responses MUST NOT include a message body.
```

Comment:
Nit: `HEAD` -> `HEAD request`  (Yeah, it's repetitive, but I think that clarity is warranted here and not particularly awkward.)

Nit: `The` -> `the`

Nit: I don't think we do ALL CAPS for emphasis in the docs. `MUST NOT` -> to `_must not_` which will render as italics.